source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -1710,6 +1682,49 @@ std::unique_ptr<OnProcessingResponse> FilterConfig::createOnProcessingResponse()
   return on_processing_response_factory_cb_();
 }
 
+void Filter::onProcessHeadersResponse(const envoy::service::ext_proc::v3::HeadersResponse& response,
+                                      absl::Status status, TrafficDirection traffic_direction) {
+  if (on_processing_response_) {
+    ASSERT(traffic_direction != TrafficDirection::UNSPECIFIED);
+    if (traffic_direction == TrafficDirection::INBOUND) {
+      on_processing_response_->afterProcessingRequestHeaders(response, status,
+                                                             decoder_callbacks_->streamInfo());
+    } else {
+      on_processing_response_->afterProcessingResponseHeaders(response, status,
+                                                              decoder_callbacks_->streamInfo());
```

Comment:
I undertand it's the same stream_callbacks_ underneath, but might worthy changing this to encode_callbacks_ so it aligns with the OUTBOUND direction. 
here and everywhere.

Reply:
Done